tests/gis_tests/gisadmin/models.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib.gis.db import models
+
+from ..admin import admin
+
+
+class City2(models.Model):
```

Comment:
This can use the same name as previous tests.
```suggestion
class City(models.Model):
```